### PR TITLE
fix: Plumb program-based subscriptions

### PR DESCRIPTION
**Pull request by CriesofCarrots**

### CODE REVIEW

File: module.flow.js [hunk 1]
```diff
@@ -76,6 +76,11 @@ declare module '@solana/web3.js' {
       callback: AccountChangeCallback,
     ): number;
     removeAccountChangeListener(id: number): Promise<void>;
+    onProgramAccountChange(
+      programIdL PublicKey,
```
Comment (mvines) [inline module.flow.js]: looks like a typeo here, unfortunately we don't currently test module.flow.js in CI and rely on downstream packages to notify of bustage 🙃

Comment (CriesofCarrots) [reply module.flow.js]: 😝

**Comment by CriesofCarrots** [comment]
@mvines I didn't realize the jest update would be critical for this PR passing CI. If you want us to merge that first and then have me rebase this, just let me know

**Comment by mvines** [comment]
Na just land it all here.

**Comment by CriesofCarrots** [comment]
@mvines , this good to go, then?

**Pull request merged**